projects/common/src/color/color.service.ts

```diff
@@ -38,11 +38,11 @@ export class ColorService {
   }
 
   public brighter(colorHex: string, basis: number): string {
-    return rgb(colorHex).brighter(basis).hex();
+    return rgb(colorHex).brighter(basis).formatHex();
   }
 
   public darker(colorHex: string, basis: number): string {
-    return rgb(colorHex).darker(basis).hex();
+    return rgb(colorHex).darker(basis).formatHex();
```

Comment:
Why this change? I dont see d3 being updated with this pr.

Reply:
This was a deprecation called out by the linter. If I were to guess, we got a new locked version of d3-color that was still compatible with the version spec in package.json